iosxr_test.py

```diff
@@ -253,11 +255,25 @@ def parse_args():
     parser.add_argument('-v', '--verbose',
                         action='store_const', const=logging.DEBUG,
                         default=logging.INFO, help='turn on verbose messages')
+    parser.add_argument('-d', '--debug', action='store_true',
+                        help='will exit with the VM in a running state. Use: "vagrant ssh" to access.')
```

Comment:
All of the help strings for this parser (and the one in iso2vbox.py, from which this is a copy-and-paste) start with lowercase letters. Just being consistent.